packages/browser/src/tracekit.ts

```diff
@@ -911,6 +912,21 @@ TraceKit._computeStackTrace = (function _computeStackTraceWrapper() {
     };
   }
 
+  function popFrames(stacktrace: any, popSize: number): any {
+    if (typeof popSize !== 'number' || popSize === 0) {
```

Comment:
My personal habit is using `isNaN` to detect number in JavaScript since `typeof NaN` also returns `number`, feel free if it's over-defend :)

Reply:
Good catch. Will change that, thanks! :)